inc/api.php

```diff
@@ -11,7 +11,7 @@
 	/**
 	 * Returns an arry of collection objects.
 	 */
-	function wm_get_collections() : array {
+	function wm_get_collections(): array {
```

Comment:
You will see this change a lot. This was done to satisfy some pre-existing PHPCS issues.